examples/models/llama/runner/eager.py

```diff
@@ -91,10 +91,11 @@ def main() -> None:
         else runner.text_completion(
             prompt=args.prompt,
             temperature=args.temperature,
+            echo=True,
```

Comment:
out of curiousity, what does `echo=True` do?

Reply:
When it is true, the returned token list will include prompt tokens as well.